ext/pdo_sqlite/sqlite_driver.stub.php

```diff
@@ -5,10 +5,10 @@
 // These are extension methods for PDO. This is not a real class.
 class PDO_SQLite_Ext {
     /** @return bool */
-    public function sqliteCreateFunction(string $function_name, callable $callback, int $num_args = -1, int $flags = 0) {}
+    public function sqliteCreateFunction(string $name, callable $callback, int $numArgs = -1, int $flags = 0) {}
 
     /** @return bool */
-    public function sqliteCreateAggregate(string $function_name, callable $step_func, callable $finalize_func, int $num_args = -1) {}
+    public function sqliteCreateAggregate(string $name, callable $step, callable $finalize, int $numArgs = -1) {}
```

Comment:
IMO it could also be

```suggestion
    public function sqliteCreateAggregate(string $name, callable $stepCallback, callable $finalizeCallback, int $numArgs = -1) {}
```

but we didn't add a postfix to callbacks so far, so I'm not doing this here.